salt/modules/hashutil.py

```diff
@@ -300,8 +289,9 @@ def github_signature(string, shared_secret, challenge_hmac):
     msg = string
     key = shared_secret
     hashtype, challenge = challenge_hmac.split("=")
-    if six.text_type:
+    if isinstance(msg, str):
         msg = salt.utils.stringutils.to_bytes(msg)
+    if isinstance(key, str):
```

Comment:
So far this is the only thing I've seen that makes me :suspect: 

Was this an automated fix or was this manual?

Reply:
Automated.